prql-compiler/src/sql/std.sql.prql

```diff
@@ -56,29 +57,33 @@ let neg = l -> null
 let not = l -> null
 
 module postgres {
-    let regex_search = text pattern -> s"{text} ~ {pattern}"
+    let div_i = l r -> s"trunc({l} / {r})::bigint"
+    let round = n_digits column -> s"ROUND(({column})::numeric, {n_digits})"
+    let regex_search = text pattern -> s"({text} ~ {pattern})"
```

Comment:
So good that we have the dialect modules now — adding something goes from difficult rust to easy text manipulation!